data/presets/leisure/schoolyard.json

```diff
@@ -0,0 +1,23 @@
+{
+    "icon": "maki-playground",
+    "fields": [
+        "name",
+        "operator",
+        "access_simple"
+    ],
+    "moreFields": [
+      "surface"
+    ],
+    "geometry": [
+        "area"
+    ],
+    "terms": [
+        "school playground",
```

Comment:
By the way, this is also known as a “blacktop” in American English (even when the asphalt pavement has faded to a light gray) to distinguish from the other meaning of “schoolyard” that refers to the whole grounds.